openassessment/xblock/test/base.py

```diff
@@ -310,6 +310,30 @@ def load_fixture_str(path):
         with open(os.path.join(base_dir, path)) as file_handle:
             return file_handle.read()
 
+    @staticmethod
+    def _create_mock_runtime(
+            item_id,
+            is_staff,
+            is_admin,
+            anonymous_user_id,
+            user_is_beta=False,
+    ):
```

Comment:
Nit: I would always vote for more kwargs so we don't have to pass the same args for most of our use cases.
```suggestion
    def _create_mock_runtime(
            item_id,
            anonymous_user_id,
            is_staff=False,
            is_admin=False,
            user_is_beta=False,
    ):
```

Reply:
I agree but in the spirit of smallest change, this was just cut-pasted from another file